actions/cloud.tsx

```diff
@@ -109,7 +109,11 @@ export function validateBusinessEmail(email = '') {
     trackEvent('api', 'api_validate_business_email');
     return async () => {
         try {
-            await Client4.validateBusinessEmail(email);
+            const res = await Client4.validateBusinessEmail(email);
+            if (res.data.is_valid === 'true') {
```

Comment:
nit: if we stick with the string here (cf review on the server PR), we should make this more robust by turning `is_valid` into lowercase?

```suggestion
            if (res.data.is_valid.toLowerCase() === 'true') {
```